tools/Utilities/IfeoTool/DevHome.IfeoTool.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(SolutionDir)ToolingVersions.props" />
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
+    <RootNamespace>DevHome.IfeoTool</RootNamespace>
+    <ApplicationManifest>app.manifest</ApplicationManifest>
+    <Platforms>x86;x64;arm64</Platforms>
```

Comment:
><Platforms>x86;x64;arm64</Platforms> [](http://example.com/codeflow?start=4&length=36)

I don't believe this line is needed

Reply:
I copied and pasted this from the HostsFileEditor.  So likely it has the same issues.  I'll remove and make sure things still work.

Reply:
fixed in latest iteration.